brave/brave-core
# Generate QR code of sync codes for mobile devices

Resolves https://github.com/brave/brave-browser/issues/10335

![image](https://user-images.githubusercontent.com/741836/86430352-e58c6d80-bca6-11ea-8a8e-b8748cf8147e.png)


![image](https://user-images.githubusercontent.com/741836/86430317-d1487080-bca6-11ea-8ec3-0d3e334c026a.png)

![image](https://user-images.githubusercontent.com/741836/86430558-75321c00-bca7-11ea-97f4-630783f4768a.png)


![image](https://user-images.githubusercontent.com/741836/86430371-efae6c00-bca6-11ea-8f50-f6252df1e970.png)

![image](https://user-images.githubusercontent.com/741836/86430539-6c414a80-bca7-11ea-92de-bfb357539fc7.png)


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Setup a sync chain on desktop
2. Choose "Phone/Tablet" to display QR code
3. Scan the QR code on Android device which has sync v2
4. Desktop and Android should be able to sync

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
